rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java

```diff
@@ -125,6 +127,10 @@ public MinerServerImpl(
         coinbaseAddress = miningConfig.getCoinbaseAddress();
         minFeesNotifyInDollars = BigDecimal.valueOf(miningConfig.getMinFeesNotifyInDollars());
         gasUnitInDollars = BigDecimal.valueOf(miningConfig.getGasUnitInDollars());
+
+        int version = 1;
+        String identity = config.projectVersionModifier() + "-" + buildInfo.getBuildHash();
+        extraData = RLP.encodeList(RLP.encodeInt(version), RLP.encodeString(identity));
```

Comment:
I would encapsulate this code on a function called something like `encodeExtraDataVersion1`

Reply:
II think it's clear enough but it costs nothing to create a private method for it